include/nodes/node.tcc

```diff
@@ -592,8 +685,12 @@ void mpm::Node<Tdim, Tdof, Tnphases>::update_property(
     unsigned nprops) noexcept {
   // Update/assign property
   node_mutex_.lock();
-  property_handle_->update_property(property, prop_id_, mat_id, property_value,
-                                    nprops);
+  if (update)
+    property_handle_->update_property(property, prop_id_, mat_id,
+                                      property_value, nprops);
+  else
+    property_handle_->assign_property(property, prop_id_, mat_id,
+                                      property_value, nprops);
```

Comment:
@bodhinandach @cgeudeker actually we might be able to remove this. Are you aware of anywhere this is used?

Reply:
Nope. This is a new addition, so I am assuming nobody else uses this. @cgeudeker Can you confirm?

Reply:
Correct